src/main/flight/pid.c

```
@@ -574,8 +574,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         pidCoefficient[axis].Kf = FEEDFORWARD_SCALE * (pidProfile->pid[axis].F / 100.0f);
     }
 #ifdef USE_INTEGRATED_YAW_CONTROL
-    if (!pidProfile->use_integrated_yaw) {
+    if (!pidProfile->use_integrated_yaw)
 #endif        
+    }
```

Inverse sign: `}`->`{`

---

Aaargh. 'You can't fail fast enough.'